extensions/wikia/DesignSystem/styles/design-system-notifications.scss

```diff
@@ -0,0 +1,153 @@
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/index';
+
+// on-site-notifications to be moved to Design System
+
+$notifications-dropdown-content-color: #656d77;
+$notifications-dropdown-border-color: #363b41;
+$wds-icon-color: #999;
+
+.wds-hidden-svg {
+  height: 0;
+  overflow: hidden;
+  position: absolute;
+  width: 0;
+}
+
+.wds-notifications {
+  &__mark-all-as-read-button {
+	box-shadow: 0 -1px 10px 0 rgba(0, 0, 0, 0.1);
+	display: none;
+	justify-content: flex-start;
+	min-height: 44px;
+	padding: 0;
+	width: 100%;
+
+	&.wds-is-visible {
+	  display: flex;
+	}
+  }
+  &__mark-all-as-read {
+	color: $wds-color-link;
+	font-size: 12px;
+	font-weight: bold;
+	line-height: 12px;
```

Comment:
Are those styles ready for longtranslations? What will happen if this message breaks to the 2nd line?